README.md

```diff
@@ -1,30 +1,38 @@
-# ezcap (@digitalbazaar/ezcap-client)
+# ezcap
 
-[![Node.js CI](https://github.com/digitalbazaar/ezcap-client/workflows/Node.js%20CI/badge.svg)](https://github.com/digitalbazaar/ezcap-client/actions?query=workflow%3A%22Node.js+CI%22)
+[![Node.js CI](https://github.com/digitalbazaar/ezcap/workflows/Node.js%20CI/badge.svg)](https://github.com/digitalbazaar/ezcap/actions?query=workflow%3A%22Node.js+CI%22)
 
-> An opinionated Authorization Capabilities (ZCAP) client library for the
-> browser and Node.js that is easy to use.
+> An easy to use, opinionated Authorization Capabilities (zcap) client library
+> for the browser and Node.js.
 
 ## Table of Contents
 
 - [Background](#background)
 - [Security](#security)
 - [Install](#install)
 - [Usage](#usage)
+- [API Reference](#api-reference)
 - [Contribute](#contribute)
 - [Commercial Support](#commercial-support)
 - [License](#license)
 
 ## Background
 
 This library provides a client that browser and node.js applications can use to
-interact with HTTP servers protected by ZCAP-based authorization. The library
+interact with HTTP servers protected by zcap-based authorization. The library
 is configured with secure and sensible defaults to help developers get started
 quickly and ensure that their client code is production-ready.
 
 ## Security
 
-TBD
+The security characteristics of this library are largely influenced by design
+that implementers make with respect to private key management. The security
+characteristics of the server that you communicate with using this library
+are largely dependent on how carefully the server manages zcap registrations,
+zcap delegations, and zcap checking. Bugs or failures related to key management,
+or enforcement of zcaps will lead to security failures. It is imperative that
+implementers audit their implementations, preferably by parties other than
```

Comment:
```suggestion
implementers audit their implementations, preferably via parties other than
```

Reply:
Fixed in fa2cf5242ea8b1e4c15feab81cccadccc62ee74e.